src/reducers/collapseContext.ts

```diff
@@ -19,7 +27,13 @@ const collapseContext = (state: State) => {
           newPath: unroot([...parentOf(cursor), child]),
         })
       ),
-      archiveThought({ path: cursor }),
+      !preventCursorArchive
+        ? archiveThought({ path: cursor }) :
+        existingThoughtDelete({
+          context: parentOf(pathToContext(cursor)),
+          showContexts: false,
```

Comment:
You can omit `showContexts` instead of explicitly setting it to false since it is an optional variable.